package.json

```diff
@@ -7,13 +7,15 @@
     "preversion": "yarn test",
     "benchmark": "node benchmark",
     "eslint": "eslint \"*.js\" benchmark test",
+    "tslint": "eslint test/typescript/index.tsx",
```

Comment:
realizing that we should probably just move to prettier. but that's for another day :)